cmd/main.go

```diff
@@ -78,19 +84,22 @@ func run(options commandOptions) error {
 }
 
 func buildCmdTree() *cobra.Command {
-	if internal.ConfiguredAndV1() {
-		internal.PrintV1DeprecationWarning()
+	if internal.IsConfigured() && !internal.IsSupportable() {
+		fmt.Printf(noticeColor, "The v1 projects are deprecated since Feb 1, 2020.\n"+
+			"See how to upgrade your project to v2: https://book.kubebuilder.io/migration/guide.html\n")
 	}
 
 	// kubebuilder
 	rootCmd := newRootCmd()
 
+	// kubebuilder update (v1 only)
+	if !internal.IsSupportable() {
+		rootCmd.AddCommand(newUpdateCmd())
+	}
+
```

Comment:
Could you leave this after init and before version? They are ordered alphabetically.

Reply:
Solved.